packages/yoastseo/src/scoring/assessor.js

```diff
@@ -221,18 +215,15 @@ Assessor.prototype.isValidResult = function( assessmentResult ) {
 };
 
 /**
- * Returns the overallscore. Calculates the totalscore by adding all scores and dividing these
+ * Returns the overall score. Calculates the total score by adding all scores and dividing these
  * by the number of results times the ScoreRating.
  *
- * @returns {number} The overallscore
+ * @returns {number} The overall score.
  */
 Assessor.prototype.calculateOverallScore  = function() {
-	var results = this.getValidResults();
-	var totalScore = 0;
+	const results = this.getValidResults();
 
-	forEach( results, function( assessmentResult ) {
-		totalScore += assessmentResult.getScore();
-	} );
+	const totalScore = results.reduce( ( total, assessmentResult ) => total + assessmentResult.getScore(), 0 );
```

Comment:
Saw the opportunity for a `reduce` (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce) and took it! 💪